src/sdk/sdk.ts

```diff
@@ -2028,6 +2028,91 @@ export class Sdk {
     return this.services.faucetService.topUpPaymentDepositAccount();
   }
 
+
+  async topUp(
+    value: string,
+  ): Promise<void> {
+    const wallet: Partial<Wallet> = this.services.walletService.walletProvider;
```

Comment:
How are we handling null here? What if walletProvider is null? Please add appropriate try/catch